src/providers/realtime-provider.js

```diff
@@ -311,4 +344,8 @@ export default class RealtimeProvider {
     isEventMessage(message) {
         return message.type === 'events';
     }
+
+    isMdbChangesMessage(message) {
+        return message.type === 'mdb-changes';
```

Comment:
can we use `DATA_TYPES.DATA_TYPE_MDB_CHANGES` here?

Reply:
There is no `OBJECT_TYPES.DATA_TYPE_MDB_CHANGES` since it's technically not an object type.